boards/litex/arty/README.md

```diff
@@ -11,21 +11,40 @@ differ significantly depending on the LiteX release and configuration
 options used. This board definition currently targets and has been
 tested with
 - [the LiteX SoC generator, revision
-  444a605dea](https://github.com/enjoy-digital/litex/tree/444a605deae6a561dbe2c49bf3062eae6f3cd887)
+  4092180662](https://github.com/enjoy-digital/litex/tree/4092180662ec62cf28b9283a020f1ff7f0892c19)
 - using the included [target
-  file](https://github.com/enjoy-digital/litex/blob/444a605deae6a561dbe2c49bf3062eae6f3cd887/litex/boards/targets/arty.py)
-- built around a VexRiscv-CPU
+  file](https://github.com/enjoy-digital/litex/blob/4092180662ec62cf28b9283a020f1ff7f0892c19/litex/boards/targets/arty.py)
+- built around a VexRiscv-CPU with PMP, hardware multiplication and
+  compressed instruction support
 - along with the following configuration options:
 
   ```
   --uart-baudrate=1000000
-  --cpu-variant=full
+  --cpu-variant=tock+secure+imc
   --csr-data-width=32
   --timer-uptime
   --integrated-rom-size=0xb000
   --with-ethernet
   ```
 
+The `tock+secure+imc` is a custom VexRiscv CPU variant, based on the
+build infrastructure in
+[pythondata-cpu-vexriscv](https://github.com/litex-hub/pythondata-cpu-vexriscv),
+using a
+[patch](https://github.com/lschuermann/tock-litex/blob/master/pkgs/pythondata-cpu-vexriscv/0001-Add-TockSecureIMC-cpu-variant.patch)
```

Comment:
Could this patch be upstreamed?

Reply:
I think there _might_ be a chance of getting this CPU configuration upstreamed in the `pythondata-cpu-vexriscv`, but based on my understanding this repository is simply a default collection of some variants anyways. When you're building some hardware / gateware, or targeting some hardware from an OS, it'd be expected for you to build some custom CPU meeting these requirements anyways. For people who don't feel like using Scala to build it, I upload the patched & built `pythondata-cpu-vexriscv` [along with every release](https://github.com/lschuermann/tock-litex/releases/tag/2020122201).